src/core/load-fragment-gperf.gperf.in

```diff
@@ -173,6 +173,7 @@
 {{type}}.SmackProcessLabel,                config_parse_warn_compat,                    DISABLED_CONFIGURATION,             0
 {% endif %}
 {{type}}.ProtectHostname,                  config_parse_bool,                           0,                                  offsetof({{type}}, exec_context.protect_hostname)
+{{type}}.MemoryKSM,                        config_parse_bool,                           0,                                  offsetof({{type}}, exec_context.memory_ksm)
```

Comment:
and then use config_parse_tristate

Reply:
I made the above change and changed the datatype of memory_ksm to int.